lrspline/raw.pyx

```diff
@@ -421,11 +438,29 @@ cdef class LRSplineObject:
     def setMaxAspectRatio(self, double r, bool aposteriori=True):
         self.w.setMaxAspectRatio(r, aposteriori)
 
-
 cdef class LRSurface(LRSplineObject):
 
-    def __cinit__(self, w=None):
-        self.w = new LRSplineSurface_()
+    def __cinit__(self, n1=None, n2=None, order_u=None, order_v=None, knot1=None, knot2=None, coef=None, dim=2):
+        cdef array.array knot1_arr;
```

Comment:
You don't need semicolons here.

Reply:
But it is C-init... you have to get into the C mindset at that point 😉 

No, I'll fix it